randovania/data/json_data/prime1/Chozo Ruins.txt

```diff
@@ -1395,12 +1397,16 @@ Asset id: 4216627403
               Any of the following:
                   Space Jump Boots
                   Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
-          Wave Beam and After Hall of the Elders Bomb Slots
+          All of the following:
+              Wave Beam and After Hall of the Elders Bomb Slots
+              Charge Beam or Combat (Intermediate)
```

Comment:
This change is unnecessary since the ghosts dont spawn when you use the wave slot, only after you reload the room. And even then, they arent required to fight and dont need to be accounted for.